Coordinator/Coordinator/Coordinators/MainCoordinator.swift

```diff
@@ -0,0 +1,29 @@
+// MainCoordinator.swift
+// Copyright © RoadMap. All rights reserved.
+
+//
+//  MainCoordinator.swift
+//  Coordinator
+//
+//  Created by Дмитрий Супрун on 28.12.22.
+// swiftlint:disable all
+
+import UIKit
+
+typealias VoidHandler = () -> ()
+
+/// MainCoordinator
+final class MainCoordinator: BaseCoordinator {
+    // MARK: - Public Properties
+
+    var rootController: UINavigationController?
+    var onFinishFlow: VoidHandler?
+
+    // MARK: - Public Methods
+
+    override func start() {
+        showMainModule()
+    }
+
+    private func showMainModule() {}
+}
```

Comment:
```suggestion
    override func start() {
        showMainModule()
    }

    private func showMainModule() {}
}
```
Что делает этот метод?